rollup-plugin-content-loader.js

```diff
@@ -124,6 +124,7 @@ export default function contentLoader() {
             content_type: contentType,
             'fields.key': key,
             limit: 1,
+            include: 10,
```

Comment:
why does the Footer branch change the Contentful rollup plugin?

@ArkaneMoose I think you're the best source for this